### DISCUSSION TITLE
Looking for the "frozen_model.pb" file for "examples\water\gmx"

### DISCUSSION THREAD

**Discussion by wxwth** [deepmodeling/deepmd-kit#3009] [Q&A] [answered]
Dear DeePMD-kit developers,
  Could you please share the "frozen_model.pb" file corresponding to the simulation in "examples\water\gmx"? It will be helpful for users to perform the exemplary simulation, and find out if there are any problems on the installation of GROMACS with DeepMD. Thanks in advance!

**Comment by njzjz** [comment, marked as answer]
You can download one from https://github.com/deepmodeling-activity/deepmd-kit-v2-paper/tree/main/models/03.

Note: it's for testing purposes instead of production.

**Reply by wxwth** [reply]
Dear Dr. Zeng,
Thank you very much for your early reply! With this file, I notice that there are some problems in my Gromacs (version 2020.2) with DeepMD (version deepmd-kit-2.2.7). It seems confusing because the Gromacs (without DeepMD) I installing can perform simulations properly, and I have trained many deep potentials using this DeepMD. I have updated the script for installing dp-gromacs, and the output files of the dp-gromacs simulation corresponding to that in "examples\water\gmx". Could you please kindly help me identify the problem? I am much obliged to you for your valuable suggestions.
(I am not sure if it is proper to raise a question in this way. If not, please point it out, I will turn to raise a issue instead.)

Script for installing dp-gromacs:
[installation_dp-gromacs-2020.2.sh.txt](https://github.com/deepmodeling/deepmd-kit/files/13465837/installation_dp-gromacs-2020.2.sh.txt)

Output files of the dp-gromacs simulation:
[md.sh.txt](https://github.com/deepmodeling/deepmd-kit/files/13465842/md.sh.txt)
[md.log](https://github.com/deepmodeling/deepmd-kit/files/13465838/md.log)
[output.9548013.txt](https://github.com/deepmodeling/deepmd-kit/files/13465840/output.9548013.txt)

**Reply by njzjz** [reply]
The error message shows an openacc error:

```
gmx: /opt/conda/conda-bld/gcc-compiler_1654084175708/work/gcc/libgomp/oacc-init.c:84: goacc_register: Assertion `!dispatchers[disp->type]' failed.
```

The source code is here: https://github.com/gcc-mirror/gcc/blob/c9d029ba2ceb435e31492c1f3f0fd3edf0e386be/libgomp/oacc-init.c#L84

It seems to me you may need to limit the [threads](https://manual.gromacs.org/documentation/current/user-guide/mdrun-performance.html#term-thread).
